components/ui/image/CustomImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Image } from 'react-native';
+
+interface CustomImageProps {
+  imageUri: string;
+  size: 'tiny' | 'small' | 'medium' | 'large' | number;
```

Comment:
in the button we use enum, we dont use also that there ?

Reply:
because it is cleaner and prittier solution, u don't have to create another Size enum that u need to import in all your code